packages/client/src/client.js

```diff
@@ -366,9 +366,9 @@ export class PercyClient {
     return snapshot;
   }
 
-  async createComparison(snapshotId, { tag, tiles = [], externalDebugUrl, ignoredElementsData } = {}) {
+  async createComparison(snapshotId, { tag, tiles = [], externalDebugUrl, ignoredElementsData, domSha } = {}) {
```

Comment:
@itsjwala  I have tried changing the `domSha` to `DOMSha` or `webDOMSha` js is converting it into `domsHa` or `webDomsHa` automatically. This breaks the E2E flow. I guess all the caps will need to be written at last.

Reply:
yes, there is a config for this check - https://github.com/percy/cli/blob/66b8192c0c24a5e16e3477d1701a201ccede4838/packages/config/src/utils/normalize.js#L5-L9 

we've a mapping which automatically updates it to given format

Reply:
I tried adding `domSha` -> `DOMSha`, it was failing.

Reply:
lets have this `domInfoSha`